Apps/VisionOS/VisionOS/Storage/AppState.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+
+struct ScreenTitleConfig {
```

Comment:
"ScreenTitleConfig" seems like it's a data type that is scoped to 1 specific screen in the app. Assuming that's correct, I think we should move the data type out of this `AppState` file.